travis/script.sh

```diff
@@ -19,7 +19,11 @@ case $BUILD in
     SRC_TGZ=$(cabal info . | awk '{print $2;exit}').tar.gz &&
       (cd dist && cabal install --force-reinstalls "$SRC_TGZ")
     ;;
-  cabal2) 
+  cabal2)
+    cabal v2-build --enable-tests --enable-benchmarks
+    cabal v2-test --enable-tests --enable-benchmarks
+    ;;
+  attoparsec-iso8601)
```

Comment:
Adding another configuration just to build `attoparsec-iso8601` seems like overkill to me. Wouldn't it suffice to just call `cabal v2-build all` in the `cabal2` configuration? Or am I missing something?

Reply:
Eh I thought there was a reason to it like this, but I think you're right.